### ISSUE TITLE
It return Error: Request did not return any results, check login information or project key

### ISSUE THREAD

**Issue by sonhb1231** [fyllepo/jira-report-time-logged-by-user#6]
![image](https://user-images.githubusercontent.com/41715305/71812856-d4bdbf00-30aa-11ea-9493-7cd42eada4d7.png)

It return Error: Request did not return any results, check login information or project key 
I have fill info in $cfg 

![image](https://user-images.githubusercontent.com/41715305/71812941-00d94000-30ab-11ea-84f0-0b25efaca07e.png)

my email is huynhbaosona2222@gmail.com

thank you so much

**Comment by fyllepo** [comment]
Hi there! 

Due to the basic nature of the error handling you'll need to debug the code a little bit your end, I'd suggest logging out the response from atlassian, then maybe it'll give some clues, and if not, post back with said response,

thanks!

**Comment by sonhb1231** [comment]
Hi Philip !
Oh.. thank you so much. I've found a bug
We must create a string with jira_email with API token, password is
incorrect, (image below)
I passed it :D

My name is Son, (21 years old)
My company is a Startup (Ecommerce + outsourcing)... In 2020 we will change
"work controll"  into Jira...
Boss assigned research about Jira , and build a tools using PHP - Laravel
... and I've found your example ...
I am having a lot of difficulties but I will try to learn and complete it!

Best regards!
[image: image.png]
